credits.js

```diff
@@ -0,0 +1,94 @@
+// Add cast listing to series page
+function addCast(series, color) {
+  let article = document.querySelector(".cast");
+  article.style.backgroundColor = color;
+  const seriesId = series.slice(29, series.indexOf("/episodes"));
```

Comment:
I'm not a fan of the 29.  it's a bit brittle, but I'm not sure there is a better way of doing it